client/state/email-accounts/actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_ACCOUNTS_REQUEST,
+	EMAIL_ACCOUNTS_REQUEST_SUCCESS,
+	EMAIL_ACCOUNTS_REQUEST_FAILURE,
+} from 'state/action-types';
+
+import 'state/email-accounts/init';
+import 'state/data-layer/wpcom/email-accounts';
+
+export const getEmailAccounts = ( siteId, useBuffer = true ) => {
+	return {
+		type: EMAIL_ACCOUNTS_REQUEST,
+		siteId,
+		useBuffer,
```

Comment:
The live http action is included, in addition to the mocked response so that we can have an idea of the shape of the endpoint.

I'll remove the mocked response and clean-up the action in another PR